pkg/pillar/hypervisor/kvm.go

```diff
@@ -54,12 +54,7 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
 [machine]
   type = "{{.Machine}}"
   dump-guest-core = "off"
-{{- if .DomainStatus.CPUs }}
```

Comment:
Why are these being removed? I think it will affect the unit tests, so you need to update them as well (same for the realtime config).... the removal reason should also be explained in the commit message...